packages/next-sanity/src/studio/client-component/useIsMounted.ts

```diff
@@ -1,3 +1,4 @@
+"use client"
```

Comment:
```suggestion
```
This directive should be lifted to the client-component/index file

Reply:
Aaah of course. I had the same hunch about maybe lifting these. Will do.